concourse/Secret.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
```

Comment:
there is no need to create this file in the github repo. you can use `kubectl create secret` when the time comes to load a secret. also, the name will have to be more specific and identifiable than `gcs-key`